packages/terra-table/src/Table.jsx

```diff
@@ -398,22 +402,26 @@ function Table(props) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [tableColumns]);
 
+  const handleResize = () => {
+    setTableHeight(tableRef.current.offsetHeight - 1);
+
+    const tableContainer = tableContainerRef.current;
+    setTableScrollable(tableContainer.scrollWidth > tableContainer.clientWidth
+                      || tableContainer.scrollHeight > tableContainer.clientHeight);
+  };
+
+  const debouncedHandleResize = LodashDebounce(handleResize, DEBOUNCE_TIMER);
+
   // useEffect for managing the table height.
   useEffect(() => {
-    const resizeObserver = new ResizeObserver(() => {
-      setTableHeight(tableRef.current.offsetHeight - 1);
-
-      const tableContainer = tableContainerRef.current;
-      setTableScrollable(tableContainer.scrollWidth > tableContainer.clientWidth
-                        || tableContainer.scrollHeight > tableContainer.clientHeight);
-    });
+    const resizeObserver = new ResizeObserver(debouncedHandleResize);
 
     resizeObserver.observe(tableRef.current);
 
     return () => {
       resizeObserver.disconnect();
     };
-  }, [tableRef]);
+  }, [debouncedHandleResize]);
```

Comment:
This logic is incorrect.  You are creating a new instance of debouncedHandleResize on every render.  So your useEffect will not trigger on every render and nullify the memoization.